mast_aladin/adapters/aladin_sync_adapter.py

```diff
@@ -9,16 +9,28 @@ def __init__(self, viewer=None):
         self.viewer = viewer if viewer else gca()
         self.aid = self.viewer.aid
 
-    def sync_to(self, sync_viewer):
-        self.aid.set_viewport(
-            **sync_viewer.aid.get_viewport(sky_or_pixel="sky")
-        )
+    def sync_to(self, sync_viewer, aspects):
+        source_viewport = sync_viewer.aid.get_viewport(sky_or_pixel="sky")
+
+        new_viewport = self.aid.get_viewport(sky_or_pixel="sky").copy()
+        if "center" in aspects:
+            new_viewport["center"] = source_viewport["center"]
+        if "fov" in aspects:
+            new_viewport["fov"] = source_viewport["fov"]
+        if "rotation" in aspects:
+            new_viewport["rotation"] = source_viewport["rotation"]
+
+        self.aid.set_viewport(**new_viewport)
 
     def add_callback(self, func):
         self.viewer.observe(func, names=["_fov", "_target", "_rotation"])
 
     def remove_callback(self, func):
-        self.viewer.unobserve(func, names=["_fov", "_target", "_rotation"])
+        for name in ["_fov", "_target", "_rotation"]:
+            try:
+                self.viewer.unobserve(func, names=[name])
+            except (ValueError, KeyError):
+                pass
```

Comment:
I think this needs to remain the same, we are adding all of the traitlets into one observation callback, so it is not correct to try to remove callbacks that only look for one of the traitlets. Was there an error that prompted this change?